### ISSUE TITLE
Black Screen of DOOOM!

### ISSUE THREAD

**Issue by FooterMan14** [EnigmaticaModpacks/Enigmatica2Expert#2076]
### Modpack Version

1.86

### Describe your issue.

When exiting a menu, the Main menu goes funky, showing the pictures of gameplay, and flicking to a black screen. It's so beautiful!

![Screenshot 2022-11-19 181501](https://user-images.githubusercontent.com/71946346/202875208-9d75bd15-2448-421e-9ba9-75409b6a7fed.png)

### Latest Log

https://gist.github.com/FooterMan14/4078ad5ef1fa4f52a6a944a5114f4cbd

### Have you modified the modpack?

Yes

### User Modifications

Added Essential.

### Did the issue happen in singleplayer or on a server?

Singleplayer

### Discord Username

✯FӨӨt🅴rMคn14#0912

**Comment by FooterMan14** [comment]
I *might* take that back, after a regular crash and loading it back up, the issue persists without going into any menu. 😆

**Comment by NielsPilgaard** [comment]
Does it happen without Essential? :)

**Comment by FooterMan14** [comment]
It does indeed happen, I just tested it. I don't know how you couldn't replicate it, if that's what you're saying.

**Comment by NielsPilgaard** [comment]
I can't reproduce it in a freshly downloaded instance, I recommend you try reinstalling :)

**Comment by FooterMan14** [comment]
Absolutely. I did forget to mention I have Opti, if that makes any difference.

**Comment by NielsPilgaard** [comment]
That could also be the culprit 😅
